protocols/v2/roles-logic-sv2/src/errors.rs

```diff
@@ -61,6 +61,7 @@ pub enum Error {
     HashrateError(InputError),
     LogicErrorMessage(std::boxed::Box<AllMessages<'static>>),
     JDSMissingTransactions,
+    MiningChannelError(u32),
```

Comment:
Why do we need to add this error type? Also, let's not prepend the variant with the word `Error` as it is redundant.

```suggestion
    MiningChannel(u32),
```